tests/system/src/test/java/test/robot/javafx/scene/TooltipTest.java

```diff
@@ -232,18 +225,17 @@ private long waitForTooltip() {
 
         assertFalse(tooltip.isShowing());
 
-        AtomicLong time = new AtomicLong();
         Util.runAndWait(() -> {
             Window window = scene.getWindow();
             robot.mouseMove(
                     window.getX() + scene.getX() + button.getLayoutX() + button.getLayoutBounds().getWidth() / 2,
                     window.getY() + scene.getY() + button.getLayoutY() + button.getLayoutBounds().getHeight() / 2);
-            time.set(System.currentTimeMillis());
+            tooltipStartTime = System.currentTimeMillis();
```

Comment:
it is better to use `System.nanoTime()` instead of `currentTimeMillis()` for this kind of measurements because it is not affected by the updates to the current time (such as time sync).

(since there are plenty of other places where we use `currentTimeMillis()` it's probably ok - very unlikely for the test to hit this scenario)

Reply:
That seems like a good overall test cleanup that could be done in a follow-up issue.